File: 프로그래머스/2/12909. 올바른 괄호/올바른 괄호.py
def solution(s):
    l = []
    
    for i in range(len(s)):
        if s[i] == "(":
            l.append(s[i])
        else:  
            if len(l) == 0:
                return False
            else:
                l.pop()

    if len(l) == 0:
        return True
    else:
        return False

File: 프로그래머스/2/12909. 올바른 괄호/올바른 괄호.js
function solution(s){
    let a = []
    if (s[0] === ")") return false 
    
    for (let i = 0; i < s.length; i++) {
        if (s[i] === "(") a.push("(")
        else a.pop()
    }
    
    return a.length === 0 ? true : false
}